website_portal_purchase/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
```

Comment:
These also do not follow the correct copyright guidelines and will all need to be changed to say `Copyright` instead of `(c)`, which further reinforces my issue with this copyright proliferation.